### PR TITLE
Update defaults

### LINKED ISSUE #30: Update default settings
Add the following:
- include Site in "Group by" control
- include Site and Marking Group as standard filters.

### PR DESCRIPTION
**Pull request by jwildfire**
fixes #30 

@samussiah - can you make sure the way I added the default filters is optimal?

### CODE REVIEW

File: src/defaultSettings.js [hunk 1]
```diff
@@ -54,25 +54,32 @@ export function syncSettings(settings) {
     groups = [
       { value_col: settings.form_col, label: "Form" },
       { value_col: "Form: Field", label: "Form: Field" },
-      { value_col: settings.status_col, label: "Status" }
+      { value_col: settings.status_col, label: "Status" },
+      { value_col: "Site Name", label: "Site" }
```
Comment (samussiah) [inline src/defaultSettings.js]: Probably should make the column name configurable or remove this group if the column doesn't exist.

File: src/defaultSettings.js [hunk 2]
```diff
@@ -135,6 +142,18 @@ export const controlInputs = [
...
   },
   {
+    type: "subsetter",
+    value_col: "Site Name", 
+    label: "Site",
+    description: "filter"
+  },
+  {
+    type: "subsetter",
+    value_col: "Query Open By: Marking Group",
```
Comment (samussiah) [inline src/defaultSettings.js]: What if the dataset doesn't contain this column?

**Review by samussiah — COMMENTED** [review]
I'm fairly reluctant to enforce a data standard that makes the chart unusable if the data don't contain two columns named _Site Name_ and _Query Opened By: Marking Group_.  I'd prefer we provide a custom settings object with the desired groupings and filters; we're already doing so with the safety explorer.

File: src/defaultSettings.js (hunk 2 above)
Comment (jwildfire) [reply src/defaultSettings.js]: Moving this to the filters option, so users can specify filters:[] to remove.

File: src/defaultSettings.js (hunk 1 above)
Comment (jwildfire) [reply src/defaultSettings.js]: Agreed. Will fix.

**Review by samussiah — APPROVED** [review]

**Pull request merged**